mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -540,6 +579,22 @@ func resourceMongoDBAtlasAdvancedClusterUpdate(ctx context.Context, d *schema.Re
 		}
 	}
 
+	/*
+		Check if advanced configuration option has a changes to update it
+	*/
+	if d.HasChange("advanced_configuration") {
+		ac := d.Get("advanced_configuration")
+		if aclist, ok1 := ac.([]interface{}); ok1 && len(aclist) > 0 {
```

Comment:
[q] any reason why you are defining `ok1` instead `ok`?

Reply:
No reason at all and good eye, will update the naming, thank you.